src/main/java/org/jmxtrans/agent/StatsDOutputWriter.java

```diff
@@ -146,9 +146,9 @@ protected synchronized String buildMetricsString(String metricName, String metri
             if (metricName.contains("kafka.connect.task-metrics.status") || metricName.contains("kafka.connect.connector-metrics.status")) {
                 String status_code;
                 switch (strValue.toLowerCase()) {
-                    case "running": status_code = "0"; break;
-                    case "unassigned": status_code = "1"; break;
-                    case "paused": status_code = "2"; break;
+                    case "paused": status_code = "0"; break;
```

Comment:
Bit late maybe, but is it worth choosing unused values so we can tell pre-patch and post-patch values apart?